public/app/plugins/panel/nodeGraph/NodeGraph.tsx

```diff
@@ -208,6 +226,26 @@ export function NodeGraph({ getLinks, dataFrames, nodeLimit, panelId, zoomMode }
         </div>
       ) : null}
 
+      {!layoutAlgorithm && (
+        <div className={styles.layoutAlgorithm}>
+          <RadioButtonGroup
```

Comment:
Could we use controls at the bottom instead? Now we have two ways of changing the algorithm 🤔 

<img width="102" alt="Screenshot 2025-03-26 at 13 36 51" src="https://github.com/user-attachments/assets/84978554-0864-47b8-9f72-56e0cc50187c" />

Reply:
I've moved those buttons to work seamlessly with the layered / force selector.

![Screenshot 2025-03-31 at 10 01 04](https://github.com/user-attachments/assets/e708bf0c-71af-4e17-8de3-b83c4c431f04)